src/ja/common/message/base.py

```diff
@@ -59,6 +59,8 @@ def _get_str_from_dict(cls, property_dict: Dict[str, object], key: str, mandator
     @classmethod
     def _get_str_list_from_dict(cls, property_dict: Dict[str, object], key: str, mandatory: bool = True) -> List[str]:
         prop = cls._get_from_dict(property_dict=property_dict, key=key, mandatory=mandatory)
+        if prop is None:
```

Comment:
I somehow think the problem here is that you request mandatory for something which isn't mandatory.

Reply:
`         uid = cls._get_str_list_from_dict(property_dict=property_dict, key="uid", mandatory=False)`
for example this, the uid list can be None and if you are to remove this if above it doesn't work.

Reply:
this worked if I use the _get_from_dict but then I have to cast and yes.